### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by bgrainger**
Fixes all the failing tests in `ApplyDeltaPackageTests` and `ApplyReleasesTests`.

### CODE REVIEW

File: src/Squirrel.Core/Utility.cs [hunk 1]
```diff
@@ -40,12 +40,20 @@ public static IEnumerable<string> GetAllFilePathsRecursively(string rootPath)
                 .Concat(Directory.GetFiles(rootPath));
         }
 
+        public static string CalculateFileSHA1(string filePath)
+        {
+            Contract.Requires(filePath != null);
+
+            using (var stream = File.OpenRead(filePath))
```
Comment (anaisbetts) [inline src/Squirrel.Core/Utility.cs]: Throw some brackets around this

**Comment by bgrainger** [comment]
Rewrote the Utility commit with better styling.

**Comment by shiftkey** [comment]
Looks good. Thanks for looking into the tests.

I've got a few Squirrel things in the queue so I'm gonna sequester myself away somewhere tomorrow and get them sorted.

**Comment by shiftkey** [comment]
There's some other headaches with IAppSetup tests which I'll look at next - but they're unrelated to this PR.

**Pull request merged**